onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -12,14 +12,25 @@
 
 class TestInferenceSession(unittest.TestCase):
 
+    def get_name(self, name):
```

Comment:
perhaps we should consistently use self.get_name() or get_name() from helper. @liqunfu thoughts?

Reply:
I talked to Liqun and he mentioned that the code that is ort_training is the correct one, so i updated the PR to only use get_name from helper.py and remove it from onnxruntime_test_python.py